src/main/java/com/amplitude/Retry.java

```diff
@@ -56,15 +56,11 @@ protected RetryEventsOnceResult(boolean shouldRetry, boolean shouldReduceEventCo
     }
 
     // Cleans up the id in the buffer map if the job is done
-    private static void cleanUpBuffer(String userId, String deviceId) {
```

Comment:
Update the cleanUpBuffer function like this, since it is only called after we remove the eventsBuffer from the idToBuffer map in retryEventsOnLoop.